e2e/cypress/integration/enterprise/ldap/ldap_group_sync_spec.js

```diff
@@ -20,6 +20,22 @@ function setLDAPTestSettings(config) {
     };
 }
 
+const resetPermissionsToDefault = () => {
```

Comment:
Would this be useful for other tests? Should it be shared?

Reply:
@saturninoabril I copied this function from another file actually. Is there a place I can put this for shared functions? I wasn't sure how that works.

Reply:
I'd suggests at `support/ui/system.js`. Maybe add two UI commands:
- `uiResetPermissionsToDefault` without `cy.apiAdminLogin()` and saving config.
- `uiSaveConfig` - we have it in several places and would be nice to add command of its own.

Also, please add typings at `system.d.ts`.

Note: `ui` prefix indicates that the action taken place within the command is done via UI flow. It may help command identification easier.

Reply:
Done thanks!

Reply:
LGTM, thanks!